packages/frontend-api/src/Resources/config/graphql-types/ModelType/Transport/TransportDecorator.types.yaml

```diff
@@ -38,8 +38,16 @@ TransportDecorator:
             payments:
                 type: "[Payment!]!"
                 description: "List of assigned payments"
+                resolve: '@=query("paymentsOfTransportQuery", value, args["displayInCartOnly"])'
+                args:
+                    displayInCartOnly:
+                        type: "Boolean"
+                        defaultValue: true
             transportTypeCode:
                 type: "TransportTypeEnum!"
                 description: "Code of transport type"
             vat:
                 type: "Vat!"
+            displayInCart:
```

Comment:
this field is still necessary?

Reply:
I think it is nice to have option to use this field